packages/libs/src/sdk/api/challenges/ChallengesApi.ts

```diff
@@ -198,6 +209,9 @@ export class ChallengesApi extends BaseAPI {
     const antiAbuseOracleEthAddress =
       await this.antiAbuseOracle.getWalletAddress()
     if (!antiAbuseOracleAttestation.result) {
+      if (antiAbuseOracleAttestation.errorCode) {
```

Comment:
I would much, much prefer an `Error` type that has the code in it that gets thrown here

I'm surprised we get here at all - feels like the `getChallengeAttestation` should fail with an HTTP error?

Reply:
https://github.com/AudiusProject/audius-protocol/pull/8514/files